IntegrationTests/AtbIntegrationTests.swift

```diff
@@ -60,8 +60,10 @@ class AtbIntegrationTests: XCTestCase {
         app.launchEnvironment = [
             "BASE_URL": "http://localhost:8080",
             "BASE_PIXEL_URL": "http://localhost:8080",
-            // usually just has to match an existing variant to prevent one being allocated and written to storage but we're changing oboarding
-            "VARIANT": "ma"
+            "DAXDIALOGS": "false",
+            "ONBOARDING": "false",
```

Comment:
These should help keep the integration tests stable even if we change onboarding and dax dialogs in future